### PR TITLE
Add 16 games; fix Quake II name inconsistency

### PR DESCRIPTION
**Pull request by mattmaniak**
Tested on M2 16 GB. Some them work out-of-the-box, some have sound issues, some needs fixes because of launching issues.

### CODE REVIEW

File: src/SUMMARY.md [hunk 1]
```diff
@@ -10,15 +10,20 @@
 - [Game Support](./game-support/README.md)
   - [Among Us](./game-support/among-us.md)
   - [Armored Core VI: Fires of Rubicon](./game-support/armored-core-6.md)
+  - [Assassin's Creed: Director's Cut Edition](./game-support/ac.md)
```
Comment (hahayupgit) [inline src/SUMMARY.md:13]: Change name to ac-directors-cut

File: src/SUMMARY.md [hunk 2]
```diff
@@ -10,15 +10,20 @@
...
   - [Battle Brothers](./game-support/battle-brothers.md)
+  - [Betrayer](./game-support/betrayer.md)
   - [Blasphemous 2](./game-support/blasphemous-2.md)
   - [Buckshot Roulette](./game-support/buckshot-roulette.md)
+  - [Call of Cthulhu](./game-support/cthulhu.md)
```
Comment (hahayupgit) [inline src/SUMMARY.md:18]: change name to call-of-cthulhu

File: src/SUMMARY.md [hunk 3]
```diff
@@ -74,8 +83,10 @@
   - [Stardew Valley](./game-support/stardew-valley.md)
   - [Star Wars Jedi: Fallen Order](./game-support/sw-fallen-order.md)
   - [Star Wars: Squadrons](./game-support/sw-squadrons.md)
+  - [Stronghold Crusader HD](./game-support/stronghold-c-hd.md)
```
Comment (hahayupgit) [inline src/SUMMARY.md:86]: change name to stronghold-crusader-hd

File: src/SUMMARY.md [hunk 4]
```diff
@@ -74,8 +83,10 @@
...
   - [Tom Clancy's Rainbow Six Siege](./game-support/tcr6s.md)
   - [The Stanley Parable: Ultra Deluxe](./game-support/tsp-ud.md)
+  - [The Vanishing of Ethan Carter](./game-support/vanishing-of-ethan.md)
```
Comment (hahayupgit) [inline src/SUMMARY.md:89]: change name to vanishing-of-ethan-carter

**Review by hahayupgit — CHANGES_REQUESTED** [review]
Looks good, change date formatting to mm-dd-yy (I prefer yy-mm-dd but this is from Isaac). Once these fixes are made and I verify the Steam IDs you should be good to merge.

**Comment by hahayupgit** [comment]
Also those name changes are just personal suggestion, I wouldn't have any issue with them as they are (except maybe Call of Cthulhu) if the Discord bot didn't rely on file names to search for a game in the Docs.

Side note but I'll probably figure out a better way to handle that as its current implementation isn't great.

**Comment by mattmaniak** [comment]
Thanks for the feedback. I have fixed all dates of the games I have added as I stick to DD/MM/YY format daily and made it by habit... Names are also updated per suggestion. I think the issue is that acronyms are encouraged in the contributing guide in README.md but there is no detailed case description/example where to acronym and where not.

**Comment by hahayupgit** [comment]
Of course! And I agree, I'm working on a better way to handle file names rn so it'll be more clarified soon

**Comment by hahayupgit** [comment]
BTW when you're done just request me for a review and I'll check it out

**Comment by mattmaniak** [comment]
Also I think that NOTEs and WARNINGs are a loosely defined too. I use to put not-gameplay-breaking info in Notes while anything that affects a gameplay in Warnings. It's my interpretation but I see that it varies from person to person and it may lower readbility.

By requesting you you mean to assign you somehow to PR or just mention? PR is ready in my side tho.

**Comment by mattmaniak** [comment]
Added 4 more games: Hellblade, Blitzkrieg 2, The Witcher 3, LEGO Clone Wars.

**Comment by mattmaniak** [comment]
Fallout 3 GOTY added.

**Review by IsaacMarovitz — APPROVED** [review]

**Pull request merged**